cinema_service/settings.py

```diff
@@ -44,6 +44,7 @@
     "django.contrib.staticfiles",
     "rest_framework",
     "rest_framework.authtoken",
+    "drf_spectacular",
     "debug_toolbar",
```

Comment:
The `debug_toolbar` should only be included in development environments. Consider wrapping its inclusion in a condition that checks if `DEBUG` is `True`.